app/client/doc/README.md

```diff
@@ -34,4 +34,44 @@ Command tree available [here](./commands/client.md)
 └── main.go                  # entrypoint
 ```
 
+## Debug Subcommands
+
+The debug command is terminal utility and set of sub-commands for rapid development and debugging of Pocket validators.
+If `debug` is run with no arguments, it drops the user into an interactive prompt session where they can trigger multiple debug message transmissions from the client binary.
+
+```bash
+$> client debug
+[...]
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Select an action:
+  ▸ PrintNodeState (broadcast)
+    TriggerNextView (broadcast)
+    TogglePacemakerMode (broadcast)
+    ResetToGenesis (broadcast)
+    ShowLatestBlockInStore (anycast)
+    MetadataRequest (broadcast)
+    BlockRequest (broadcast)
+
+```
+
+If it's run with an argument it selects the message type that matches that argument.
+The accepted command structure and available sub-commands are shown below.
+
+```bash
+Usage:
+  client debug [flags]      <-- drops you into a debug command prompt
+  client debug [command]    <-- accepts sub commands 
+
+Available Commands:
+  BlockRequest
+  MetadataRequest
+  PrintNodeState
+  ResetToGenesis
+  ShowLatestBlockInStore
+  TogglePacemakerMode
+  TriggerNextView
```

Comment:
I appreciate the straightforward approach to mapping the existing interactive menu out to CLI sub-commands. However, I think there are some UX considerations that we should consider:

1. Am I correct in assuming that these will be case sensitive?
2. The interactive menu is hierarchically flat  and uses multiple words in the entries to communicate their context. This is in contrast to a conventional CLI which I would expect to nest sub-commands hierarchically such that the context is provided by the hierarchy (e.g. `wasmd query account <address>` instead of `wasmd QueryAccount <address>)

Wdyt about renaming and/or restructuring the sub-commands to simplify the names and express a more conventional UX (when compared to cosmos-sdk based CLIs, for example)?

For example (respective ordering):
```
query block
query metadata  (or just `meta`)
consensus print
consensus reset
? (I'm not familiar with this command)
consensus pacemaker
consensus next
```

I acknowledge that this is probably out of scope for this PR and would be more than happy to see a new issue and a comment referencing it for the time being.

Reply:
1. Yes, I believe they're case sensitive. 
2. I did consider custom-mapping all of these to sub-commands. The reason I chose this approach was because expansion seems likely, as we already have plans to add more commands, and I wanted them to maintain the exact same wording so that it was clear they were the same commands. I worried that if we broke it out into completely new sub-commands that it wouldn't be as clear. Future extension of these commands is made easier by nature of the interactive prompt being automatically maintained, too. 

🤔 On the other hand, the categorization of consensus, query, etc... is tidy and I can't argue there. In my head, it makes sense for them to be more specific and customized since they're not part of the production CLI.

Reply:
Created #708 to cover the further changes around sub-command organization that I discussed with @bryanchriswhite .